src/netbase.cpp

```diff
@@ -3,6 +3,10 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
```

Comment:
@achow101 Its very mysterious, when I rebase this branch on master (`git rebase master`) these lines **disappear.** I can't figure out why!

However when I do interactive `git rebase master -i` and stop on this commit with an `edit` command, DO NOTHING, and just `git rebase --continue`... the lines remain intact. I am force-pushing this interactive rebase branch now.  :face_with_spiral_eyes: